### PR TITLE
Updated setup and next steps

### PR DESCRIPTION
**Pull request by rachelappel**
Partial fix 2119

### CODE REVIEW

**Comment by rachelappel** [comment]
@sayedihashimi Want to take a quick look at the setup and next steps changes here?

File: aspnetcore/tutorials/publish-to-azure-webapp-using-vs.md [hunk 1]
```diff
@@ -16,11 +16,9 @@ uid: tutorials/publish-to-azure-webapp-using-vs
 
 By [Rick Anderson](https://twitter.com/RickAndMSFT), [Cesar Blum Silveira](https://github.com/cesarbs), and [Rachel Appel](https://twitter.com/rachelappel)
 
-## Set up the development environment
+## Set up
 
-* Install the latest [Azure SDK for Visual Studio](https://www.visualstudio.com/vs/azure-tools/). The SDK installs Visual Studio if you don't already have it.
-
-* Verify your [Azure account](https://portal.azure.com/). You can [open a free Azure account](https://azure.microsoft.com/pricing/free-trial/) or [Activate Visual Studio subscriber benefits](https://azure.microsoft.com/pricing/member-offers/msdn-benefits-details/).
+* Open a [free Azure account](https://azure.microsoft.com/en-us/free) if you do not have one.
```
Comment (sayedihashimi) [inline aspnetcore/tutorials/publish-to-azure-webapp-using-vs.md]: Can we get this updated to include the tracking query string parameter? For example see https://docs.microsoft.com/en-us/azure/app-service/app-service-web-get-started-dotnet#prerequisites

**Comment by sayedihashimi** [comment]
I left a comment regarding the free azure link, otherwise this looks good.

**Comment by rachelappel** [comment]
@sayedihashimi Can we reuse the one from that article or do we need a specific one for this article?

**Comment by sayedihashimi** [comment]
@rachelappel I’m not sure. I think it would be best to check with the person who created that ID and how it’s going to be reported.

**Comment by sayedihashimi** [comment]
Ideally we could have a unique ID for each doc so we can see the impact of each doc.

**Comment by Rick-Anderson** [comment]
@rachelappel  do we need to escalate getting the id?

**Comment by rachelappel** [comment]
@Rick-Anderson Got one, it's in the doc. Going to merge.

**Pull request merged**